package practice;

public class LearnException {
    public void testException() {
            int[] num = {1, 2, 3, 4};
            System.out.println(num[5]);
    }

    public void sum() {
        System.out.println(1+2);
    }

    public void multiplication() {
        System.out.println(1*2);
    }
}
